src/ui/link/test.tsx

```diff
@@ -29,173 +30,154 @@ const defaultProps = {
   href: '/my-link',
 };
 
-const baseClickEvent = {
-  preventDefault: jest.fn(),
-  button: 0,
-};
-
 const newPath = '/my-new-path';
-const eventModifiers = [['metaKey'], ['altKey'], ['ctrlKey'], ['shiftKey']];
-
-// https://github.com/facebook/jest/pull/5267#issuecomment-356605468
-const withoutConsoleError = (fn: () => void) => () => {
-  const consoleError = jest
-    .spyOn<Console, 'error'>(console, 'error')
-    .mockImplementation(() => undefined);
-  fn();
-  consoleError.mockRestore();
-};
+// const eventModifiers = [['metaKey'], ['altKey'], ['ctrlKey'], ['shiftKey']];
+const eventModifiers = ['ShiftLeft', 'ControlLeft', 'AltLeft', 'MetaLeft'];
 
 describe('<Link />', () => {
-  const mountInRouter = (
+  const renderInRouter = (
     children: LinkProps['children'],
     props: Partial<LinkProps> = defaultProps,
     basePath = ''
-  ) =>
-    mount(
+  ) => {
+    return render(
       // @ts-expect-error
       <Router basePath={basePath} history={HistoryMock} routes={[]}>
         <Link {...props}>{children}</Link>
       </Router>
     );
+  };
 
   afterEach(() => {
     jest.resetAllMocks();
     defaultRegistry.stores.clear();
   });
 
   it('should render a <Link />', () => {
-    const wrapper = mountInRouter('my link');
-    const anchor = wrapper.find('a');
-
-    expect(anchor.prop('href')).toEqual(defaultProps.href);
-  });
-
-  it('should support the `to` prop', () => {
```

Comment:
Why did we remove this?

Reply:
Sorry for my mistake. So many tests in this file and they need a big re-work from enzyme to RTL. Added back in